config/configmap/inferenceservice.yaml

```diff
@@ -72,7 +72,8 @@ data:
         "memoryRequest": "100Mi",
         "memoryLimit": "1Gi",
         "cpuRequest": "100m",
-        "cpuLimit": "1"
+        "cpuLimit": "1",
+        "defaultUrl": "http://default-broker"
```

Comment:
So this will require the knative eventing install right?

Reply:
It defaults to looking for a knative broker but  it will just result in a warning if none is there.  It’s actually the same as current as right now it uses “default-broker.namespace”, which is the same as “default-broker” since having the current namespace in a service address is optional in k8s